crates/bevy_pbr/src/render/pbr_bindings.wgsl

```diff
@@ -23,3 +23,11 @@
 @group(2) @binding(17) var diffuse_transmission_texture: texture_2d<f32>;
 @group(2) @binding(18) var diffuse_transmission_sampler: sampler;
 #endif
+#ifdef PBR_MULTI_LAYER_MATERIAL_TEXTURES_SUPPORTED
+@group(2) @binding(19) var clearcoat_texture: texture_2d<f32>;
+@group(2) @binding(20) var clearcoat_sampler: sampler;
+@group(2) @binding(21) var clearcoat_roughness_texture: texture_2d<f32>;
+@group(2) @binding(22) var clearcoat_roughness_sampler: sampler;
+@group(2) @binding(23) var clearcoat_normal_texture: texture_2d<f32>;
+@group(2) @binding(24) var clearcoat_normal_sampler: sampler;
```

Comment:
The continued per-texture sampler is rough, but I don't think there's a better solution until we have bindless and can use indices into a small array of samplers.

Reply:
Yeah, I really don't like it either. (In general glTF may have gone overboard with all the textures at the standards level; if you're really getting that exact about e.g. clearcoat roughness you probably want a node graph like MaterialX anyway. But standards are standards, so I added support anyway...)